core/common/src/main/java/alluxio/underfs/options/CreateOptions.java

```diff
@@ -54,6 +57,13 @@ private CreateOptions() {
     mMode = Mode.defaults().applyFileUMask();
   }
 
+  /**
+   * @return the Acl
```

Comment:
comment this can be `null`, and what that means

Reply:
done